src/commands/translate/translate.command.js

```diff
@@ -69,16 +100,19 @@ const fixTranslation = translation => {
 };
 
 const handler = async message => {
+  const isSlash = message instanceof CommandInteraction;
```

Comment:
hmm. this doesn't quite match the idea I gave in my refactor comment [here](https://github.com/gganon/habla-bot/pull/28/files/d405161e64fcb19b16a1c592225ac12afd5b5b52#diff-ad98bc315862880024b07faaa7402e291ff7537c00d2c7c474adb28d6ba53447R122)

I was okay with there being 2 handler functions like you had previously: one for slash commands (`interactionHandler()`) and one for text commands (`handler()`). I was only suggesting you take the common parts out into a separate function and then let the code inside both handlers call that common function.

What you've done here is make a single `handler()` function that is trying to be flexible by checking the type of `message` and slightly changing its behavior according to that. While this seems okay for now, this can become hard to maintain because this `handle` function would always need to have 2 types of code inside it: one type of code for dealing with message text and one type of code for dealing with an interaction and switching between them based on the type of the argument.

Take a look at the comment I linked above again and see if the pseudo code makes sense. Feel free to ping me on discord if you're confused, I can give you a less "pseudo-code"-ey explanation